test/README

```diff
@@ -3,33 +3,49 @@
 This directory contains automated integration tests for Cockpit, and the support
 files for them.
 
-To run the tests on Fedora, you need to install the following packages:
-
-    $ sudo yum install npm qemu-kvm python \
-         firewalld libvirt-daemon libvirt-client libvirt-python \
-         openssl expect rsync curl xz
+To run the tests on Fedora, refer to the [HACKING](../HACKING.md) guide for
+installation of all of the necessary build and test dependencies. There's
+no need to trigger a build manually - the test suite preparation step below
+will handle that.
 
 You may need to reboot after installing the above packages. For initialization
 the following command needs to be run as root to setup the libvirt bridged network:
 
     $ sudo ./test/vm-prep
 
+Note that this step also attempts to ensure nested virtualization is enabled,
+and may emit a spurious `rmmod: ERROR: Module kvm_amd is not currently loaded`
+warning when running on an Intel CPU (or vice-versa).
```

Comment:
I created a separate PR for this: https://github.com/cockpit-project/cockpit/pull/5981

So I'll drop this paragraph from the README update.